packages/@lwc/ssr-compiler/src/compile-js/lwc-import.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { builders as b } from 'estree-toolkit';
+
+import type { ImportDeclaration } from 'estree';
+import type { NodePath } from 'estree-toolkit';
+import type { ComponentMetaState } from './types';
+
+export function replaceLwcImport(path: NodePath<ImportDeclaration>, state: ComponentMetaState) {
```

Comment:
It does two things:

1. it replaces `"lwc"` with `"@lwc/ssr-runtime"` in an import specifier
2. it keeps track of the local variable name associated with `LightningElement`. It is probably `LightningElement`. But this would be valid:

```javascript
import { LightningElement as Rumplestiltskin } from 'lwc';
export default class Foo extends Rumplestiltskin {}
```